Demo/Demo/DemoSettings/DemoType.swift

```diff
@@ -11,11 +11,11 @@ enum DemoType: String {
     var swiftUIView: some View {
```

Comment:
This can instead be:

```
    var swiftUIView: any View {
        switch self {
        case .card:
            return CardPaymentView()
        case .cardVault:
            return CardVaultView()
        case .payPalWebCheckout:
            return PayPalWebView()
        case .paymentButtonCustomization:
            return SwiftUIPaymentButtonDemo()
        case .payPalNativeCheckout:
            return SwiftUINativeCheckoutDemo()
        }
```

To avoid the cast to `AnyView`

Reply:
You know I realized we don't need this. It was for toggling settings menu for different features. It is now only referenced in CardUITests and I don't think we can use test since we are not using Demo type in UserDefaults anymore. This test doesn't seem to be run in the pipeline.

Reply:
Yes, I had to double check. This is not being used anymore since we are not toggling DemoType in userDefaults anymore. We are just controlling the navigation upon selection to the feature views. This DemoType is referenced in CardUITests but I think it wasn't working for SwiftUI anyway and not used in pipeline.

Reply:
Yup, the existing UITests will be totally broken at this point, since the Demo app has changed so much. We definitely need a ticket to revamp those, or remove them entirely. I vote for the later at this time.

Reply:
Addressed here: 2dadeba3c04086abbf65c9fbcd514bd7ef88791d